docs/manuals/gui/viselements/charts/candlestick-charts.md

```diff
@@ -0,0 +1,257 @@
+# Candlestick Chart
+
+A variation of a bar chart in which the open and close prices are represented as the top and bottom of a rectangle surrounding a vertical line connecting the high and low prices.
+
+## Candlestick Chart Key Parameters <a id="table"></a>
+Explanation of parameters and options relevant to candlestick charts can be found [here](#table). Full details for the Taipy Chart control can be found in the Chart Control section of the [Taipy GUI User Manual](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/)
+
+| Parameter       | Value Options             | Location           |
+| --------------- | ------------------------- | ------------------ |
+| Type          | "candlestick"          | Control definition |
+| high,low,open,close | Values to plot | Control definition |
+| rangeslider | dict of parameters in axis layout | layout |
+
+## Required Libraries
```

Comment:
Needs to indicate that this is the required libraries for the demo below, not a required libraries for candlestick charts to work